micro-rdk/src/esp32/conn/network.rs

```diff
@@ -268,12 +270,31 @@ impl Esp32WifiNetwork {
         &self,
         mut networks: Vec<NetworkSetting>,
     ) -> Result<(), WifiManagerError> {
-        // TODO(RSDK-10184): scan available networks first
-        // attempt to connect only if available
+        log::info!("initializing wifi module");
+        // `set_station_mode` ends by calling `connect`, which will error here
+        let _err = self
```

Comment:
I don't like the idea of silencing the error. If something else goes wrong then may be hard to track down the root cause.

The flow of attempting to connect to WiFi with an empty ssid is flawed imo. We have no guarantee this will continue to work moving forward